.github/workflows/integrate.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         provider: microk8s
         channel: 1.24/stable
         # Pinned until this bug is resolved: https://bugs.launchpad.net/juju/+bug/1992833
-        bootstrap-options: "--agent-version=2.9.34"
+        bootstrap-options: "--agent-version=2.9.34 --model-default update-status-hook-interval='30s'"
```

Comment:
I understand we decrease update interval to speed up test. My concern about not using standard, real-life timing of 5 minutes comes from MySQL integration work. In that case, workload that talks to the DB was started without DB config or with incorrect DB config and timed out at 2-3 minutes and update status at 5 mins interval properly caught this situation. In this case, can we confirm workload behaviour: timeout is not happening/crash is not happening beyond 30 seconds interval.